doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1031,8 +1031,9 @@ Controlling display
 
 .. flag:: Printing Dependent Evars Line
 
-   This option controls the printing of the “(dependent evars: …)” line when
-   ``-emacs`` is passed.
+   This option controls the printing of the “(dependent evars: …)” information
+   after each tactic.  The information is used by the Prooftree tool in Proof
+   General. (https://askra.de/software/prooftree)
```

Comment:
Why remove the info that this is used when the `-emacs` flag is passed?

Reply:
The code no longer requires `-emacs` to see the variables.  Originally -emacs alone triggered the printing before there was the option.  Once you have the option the -emacs is not needed.  So just correcting the doc, not something I changed.